sleap/gui/app.py

```diff
@@ -1380,7 +1383,12 @@ def plotFrame(self, *args, **kwargs):
 
         self.player.plot()
 
-    def _after_plot_update(self, player, frame_idx, selected_inst):
+    def _after_plot_update(self, frame_idx):
+        """Run after plot is updated, but stay on same frame."""
+        overlay: TrackTrailOverlay = self.overlays["trails"]
+        overlay.redraw(self.state["video"], frame_idx)
```

Comment:
We only use this for a single scenario right now (trails), but it could be scaled up in the future as we see necessary.